src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/mapper/LimitedPartnershipPatchMapperTest.java

```diff
@@ -82,10 +84,12 @@ private DataDto createMongoDto() {
     private void checkExpectedFieldValues(DataDto mongoDto,
                                           String expectedPartnershipName,
                                           String expectedEmail,
-                                          String expectedJurisdiction) {
+                                          String expectedJurisdiction,
+                                          String expectedPartnershipType,
+                                          String expectedPartnershipNameEnding) {
         assertEquals(expectedPartnershipName, mongoDto.getPartnershipName());
-        assertEquals(PartnershipNameEnding.L_DOT_P_DOT.getDescription(), mongoDto.getNameEnding());
-        assertEquals(PartnershipType.PFLP, mongoDto.getPartnershipType());
+        assertEquals(expectedPartnershipNameEnding, mongoDto.getNameEnding());
+        assertEquals(expectedPartnershipType, mongoDto.getPartnershipType().toString());
```

Comment:
Maybe not changed in this PR so just an observation - calling toString() on one enum but but not another looks a bit odd. Suggests we have different implementations of enums.

Reply:
does the getPartnershipType need to return a string like getNameEnding in the DataDto?